integration-tests/Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let linkerSettings: [LinkerSetting] = [
+    .linkedLibrary("swift_integration_tests"),
+    .unsafeFlags(["-L../target/debug/"])
```

Comment:
Can you see if there is a way to make this conditionally use debug or release based on an environment variable?

This makes it easier for someone to look at this script and adapt it to their own application's needs.

We can default to debug.

Reply:
Maybe, but in general this stems from the problem that creating a SPM package from a swift-bridge crate requires a lot of different bits of configuration to match up. I will explicate that in a new issue (also talking about the imports).

Reply:
(#315)